R/rstudioapi_util.R

```diff
@@ -2,7 +2,13 @@ rstudioapi_call <- function(action, ...) {
     request_response("rstudioapi", action = action, args = list(...))
 }
 
+rstudioapi_enabled <- function() {
+    isTRUE(getOption("vsc.rstudioapi"))
```

Comment:
Does it make sense that the option check be moved to `init.R` to enclose https://github.com/Ikuyadeu/vscode-R/pull/422/files#diff-1e8c7b94d7dc2f0d70c4e457124889b5637ba42993015643e5d80be52e3ec912R432-R478 so that if `vsc.rstudioapi` is not `TRUE` then the entire rstudioapi-related code won't be excuted.

Reply:
I think so. Still needed in the updateAddin registry call, because that's called in .vsc.attach or we can wrap around it in there?

Reply:
Although the response mechanism does not have to only be used by the rstudioapi?

Reply:
I guess we can move the guard if we want to use it for something else.

Reply:
`.vsc.attach` should check `rstudioapi_enabled()` before updating addin registry.

We could single out the response mechanism in the future if we use it to something else.

Reply:
okay done

Reply:
nope sorry missed a change!

Reply:
Ok now done